.gitignore

```diff
@@ -45,6 +46,10 @@ android/keystores/debug.keystore
 
 # credentials
 .npmrc
+google-services.json
```

Comment:
We found a way to commit these files without secret so you can remove this line.
Please refer to https://github.com/Microsoft/AppCenter-SDK-React-Native/blob/develop/TestApp/android/app/google-services.json if you use the same package name in the ts test app then you can just use the same file.

Reply:
Weird that it didn't get merged. What is the google-services used for, and how do I make sure that works besides just adding it as a build dependency?

Reply:
Like in TestApp, make sure firebase is integrated: https://docs.microsoft.com/en-us/appcenter/sdk/push/migration/react-native-android. 

When the procedure asks to download google-services.json, pick https://github.com/Microsoft/AppCenter-SDK-React-Native/blob/develop/TestApp/android/app/google-services.json instead and make sure your app has the same package name as TestApp.